tests/unit/Types/ContextFactoryTest.php

```diff
@@ -14,11 +14,14 @@
 
 // Added imports on purpose as mock for the unit tests, please do not remove.
     use \ReflectionClass;
```

Comment:
Can you move this line to come right after the PHPUnit grouping?  That way, the parsing rules will have a line after the grouping to process.  I'd like to see how the test goes after that change.